src/app/accounting/migrate-opening-balances/migrate-opening-balances.component.html

```diff
@@ -80,15 +80,15 @@
 
             <div fxFlexFill fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column" [formGroupName]="i">
 
-              <div fxFlex="8%">
+              <div fxFlex="8%" style="padding-top: 31px;">
                 {{ (openingBalancesData.glAccounts[i].glAccountType.value !== openingBalancesData.glAccounts[i-1]?.glAccountType.value) ? openingBalancesData.glAccounts[i].glAccountType.value : '' }}
               </div>
 
-              <div fxFlex="18%">
+              <div fxFlex="18%" style="padding-top: 31px;">
                 {{ openingBalancesData.glAccounts[i].glAccountCode }}
               </div>
 
-              <div fxFlex="18%">
+              <div fxFlex="18%" style="padding-top: 31px;">
```

Comment:
Instead of using inline styles, I would suggest using a class here and declare the styles in scss file.

Reply:
> Instead of using inline styles, I would suggest using a class here and declare the styles in scss file.

Sure. I'll fix it